proto/tripcommon.proto

```diff
@@ -38,10 +38,12 @@ message Location {
     optional LatLng ll = 3;
     optional SideOfStreet side_of_street = 4;
     optional float distance = 5;
-    optional int32 minimum_reachability = 6;
+    optional int32 minimum_reachability = 6; //deprecated
```

Comment:
now we have directional reachability so we no longer support a single non-sensical measure